spec/commands/exec_spec.rb

```diff
@@ -61,7 +61,7 @@
   it "handles --keep-file-descriptors" do
     require "tempfile"
 
-    bundle_bin = File.expand_path("#{Spec::Path.bin}/bundle", __FILE__)
+    bundle_bin = File.expand_path(bindir.join("bundle"), __FILE__)
```

Comment:
no need to use `File.expand_path` when you already have a `Pathname`. Additionally, `bindir` is absolute, so just `bindir.join("bundle")` should do the trick here

Reply:
(same goes for the rest of the specs, really)

Reply:
Make sense. I remove `File.expand_path` from a needless location.